### ISSUE TITLE
Single program

### ISSUE THREAD

**Issue by git-sumana** [saeed-rhimi/snakeGame#1]
i can embed both the scoreboard and the game together

**Comment by saeed-rhimi** [comment]
Yes, it can definitely be done, but it is better not to do it. In this way, making changes and maintaining the code will increase

**Comment by git-sumana** [comment]
> Yes, it can definitely be done, but it is better not to do it. In this way, making changes and maintaining the code will increase

can you assign it to me? I shall give a try!
